docs/docs-content/integrations/community_packs.md

```diff
@@ -0,0 +1,73 @@
+---
+sidebar_label: "Community Packs"
+title: "Community Packs"
+description: "Identify Community Packs in Palette, which are not actively maintenance."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 30
+tags: ["packs", "community"]
+---
+
+Palette Community Packs are packs that are created and maintained by the Spectro Cloud community. They provide important
+integrations with Spectro Cloud Palette.
```

Comment:
We could also mention something about the update frequency. From the repo README:
  _Typically you should get a response within 72 hours of raising your pull request. Make sure you watchout for any review comments and handle them.
  Once your PR is merged, it will be deployed to the Spectro Cloud public pack registry.
  Frequency of pushing to the registry is biweekly and happens on Tuesdays and Thursdays_.

Reply:
I had mentioned the contribution guidelines in Next Steps and Default registries pages further down in the document, but moved everything up to intro as suggested. Looks good!

@caroldelwing In my opinion, it is too much detail to add about the pack release details, so I've not added this yet.

Reply:
regarding update frequency, one of the main differences with verified packs is that spectro cloud does not actively maintain every pack in the community repository. therefore, updates to packs in that repository are dependent on the contributors from the community, and so, if people want them to be updated or want to know more about them, they may need to contact the main contributors directly. for example, the kubecost pack.

Reply:
Cool! I've added this information to the warning box located at the end of the section.